commit 40588629ff123143e3f3e143f9a139cd1f763dcf (refs/changes/13/1813/6)
Author: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date:   2017-06-23T17:12:30+02:00 (2 years, 4 months ago)
    
    OS-6067 tools/build_illumos fails if run from tcsh
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/tools/build_illumos b/tools/build_illumos
index c362cb80..7f493080 100755
--- a/tools/build_illumos
+++ b/tools/build_illumos
@@ -11,7 +11,7 @@
 #
 
 #
-# Copyright 2016 Joyent, Inc.
+# Copyright (c) 2017 Joyent, Inc.
 #
 
 # Setup for building stuff!
@@ -19,6 +19,7 @@ unset CC
 unset CXX
 ROOT=`pwd`
 export PATH=/opt/onbld/bin:/sbin:/usr/sbin:/usr/bin:/usr/ccs/bin:/opt/SUNWspro/bin:/opt/local/bin
+export SHELL=/usr/bin/bash
 
 mkdir -p ${ROOT}/proto
 if [[ ! -f ${ROOT}/proto/buildstamp ]]; then
